packages/cactus-plugin-satp-hermes/src/main/typescript/core/stage-services/client/stage0-client-service.ts

```diff
@@ -270,6 +292,20 @@ export class Stage0ClientService extends SATPService {
       getHash(preSATPTransferRequest),
     );
 
+    await storeLog(
+      {
+        sessionID: sessionData.id,
+        type: String(MessageType.PRE_SATP_TRANSFER_REQUEST),
+        operation: "create",
+        data: safeStableStringify(sessionData),
```

Comment:
For example, in the begining of this function we should have an "exec-String(MessageType.PRE_SATP_TRANSFER_REQUEST)" log meaning it will start executing it; after correct execution persist a log done-String(MessageType.PRE_SATP_TRANSFER_REQUEST)". in case of error (use try catch), persist fail-String(MessageType.PRE_SATP_TRANSFER_REQUEST)"